src/App.tsx

```diff
@@ -190,7 +190,18 @@ export const App: React.FC = () => {
           <SplashScreen setSplashScreen={handleSetSplashScreen} />
         </div>
       )}
-
+      {!isSplashScreen && (
+        <div className="reset_orbit_selection_container">
+          <div
+            className="reset_selection top"
+            onClick={resetOrbitSelection}
+          ></div>
+          <div
+            className="reset_selection bottom"
+            onClick={resetOrbitSelection}
+          ></div>
+        </div>
+      )}
```

Comment:
This is what I added . It is a completely separate div that is `position: absolute` so it takes up no space and just sits over everything minus the nav menu and popups